suite-native/message-system/src/components/MessageScreen.tsx

```diff
@@ -0,0 +1,110 @@
+import { Linking } from 'react-native';
+
+import { IconName } from '@suite-common/icons/src';
+import { Box, Button, Pictogram, PictogramVariant, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Message, Variant } from '@suite-common/suite-types';
+
+type MessageScreenProps = {
+    message: Message;
+    defaultTitle?: string;
+    defaultContent?: string;
+    secondaryButtonLabel?: string;
```

Comment:
I think we should force to define both `secondaryButtonLabel` and `onSecondaryButtonPress` at the same time or none of them. Now only one of these props can be defined individually and I think that is wrong. Please use `RequireAllOrNone` from type fest, to fixt this. You can find inspiration in https://github.com/trezor/trezor-suite/blob/33c6ca58bf1c60c7cf1c503e9003cb0ed76bbf09/suite-native/alerts/src/alertsAtoms.ts, `type Alert` specifically.